simulariumio/data_objects/model_meta_data.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Any
+import logging
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class ModelMetaData:
+    title: str
+    version: str
+    authors: str
+    description: str
+    doi: str
+    source_code_url: str
+    source_code_license_url: str
+    input_data_url: str
+    raw_output_data_url: str
+
+    def __init__(
+        self,
+        title: str = None,
+        version: str = None,
+        authors: str = None,
+        description: str = None,
+        doi: str = None,
+        source_code_url: str = None,
+        source_code_license_url: str = None,
+        input_data_url: str = None,
+        raw_output_data_url: str = None,
+    ):
+        """
+        This object holds metadata for a model that generates simulation trajectories
+        Parameters
+        ----------
+        title : str (optional)
+            Display title for this model
+        version : float (optional)
+            Version number of the model that produced this trajectory
+        authors : str (optional)
+            Modelers name(s)
+        description : str (optional)
+            Comments to display with the trajectories generated by this model
+        doi : str (optional)
+            The DOI of the publication accompanying this model
```

Comment:
Whichever is decided (URL, ID, or both), it would be good to note it in this comment